openapi-generator/src/main/resources/templates/kotlin-micronaut/client/auth/Authorization.mustache

```diff
@@ -0,0 +1,31 @@
+{{>common/licenseInfo}}
+package {{invokerPackage}}.auth
+
+import io.micronaut.core.bind.annotation.Bindable
+
+import kotlin.annotation.AnnotationRetention.RUNTIME
+import kotlin.annotation.AnnotationTarget.FUNCTION
+{{#generatedAnnotation}}
+import {{javaxPackage}}.annotation.Generated
+{{/generatedAnnotation}}
+
+{{#generatedAnnotation}}
+{{>common/generatedAnnotation}}
+{{/generatedAnnotation}}
+@MustBeDocumented
+@Retention(RUNTIME)
+@Target(FUNCTION)
+@Bindable
+@Repeatable
+annotation class Authorization(
```

Comment:
in general I am wondering why we don't upstream more of this annotation/filter logic

Reply:
I don't know, that's what was in the java generator. I just converted this to kotlin